<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>862</width>
    <height>616</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QHBoxLayout" name="globalLayout" stretch="1,0">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="imageLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>1</width>
            <height>1</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel { background-color : black; }</string>
          </property>
          <property name="text">
           <string>imageLabel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QScrollArea" name="scrollArea">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOn</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustIgnored</enum>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents_2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>208</width>
           <height>595</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QGroupBox" name="previsualisationBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="title">
             <string>Option de pr√©visualisation</string>
            </property>
            <property name="checkable">
             <bool>false</bool>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QGroupBox" name="avApBox">
               <property name="title">
                <string>Avant/Apres correction</string>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                 <widget class="QRadioButton" name="AvantButton">
                  <property name="text">
                   <string>Avant</string>
                  </property>
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QRadioButton" name="ApresButton">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="text">
                   <string>Apres</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="gainLayout">
               <item>
                <widget class="QLabel" name="gainLabel">
                 <property name="text">
                  <string>Gain</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSlider" name="gainSlider">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimum">
                  <number>-3000</number>
                 </property>
                 <property name="maximum">
                  <number>3000</number>
                 </property>
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="masqueLayout">
               <item>
                <widget class="QLabel" name="paramMasqueLabel">
                 <property name="text">
                  <string>Paramertes Masque</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="Line" name="lineMasque">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QGroupBox" name="affMasqueBox">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="title">
                <string>Afficher le Masque</string>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_3">
                <item>
                 <widget class="QComboBox" name="rayonAngleComboBox">
                  <item>
                   <property name="text">
                    <string>Rayon Angulaire (en deg)</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Distance + Rayon</string>
                   </property>
                  </item>
                 </widget>
                </item>
                <item>
                 <layout class="QFormLayout" name="rayMasqueLayout">
                  <item row="0" column="0">
                   <widget class="QLabel" name="labelAngle">
                    <property name="text">
                     <string>Angle</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <widget class="QDoubleSpinBox" name="angleBox">
                    <property name="maximum">
                     <double>180.000000000000000</double>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="0">
                   <widget class="QLabel" name="labelDist">
                    <property name="enabled">
                     <bool>false</bool>
                    </property>
                    <property name="text">
                     <string>Dist.</string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="1">
                   <widget class="QDoubleSpinBox" name="distBox">
                    <property name="enabled">
                     <bool>false</bool>
                    </property>
                    <property name="maximum">
                     <double>10000000000000000369475456880582265409809179829842688451922778552150543659347219597216513109705408327446511753687232667314337003349573404171046192448274432.000000000000000</double>
                    </property>
                   </widget>
                  </item>
                  <item row="2" column="0">
                   <widget class="QLabel" name="labelRay">
                    <property name="enabled">
                     <bool>false</bool>
                    </property>
                    <property name="text">
                     <string>Ray.</string>
                    </property>
                   </widget>
                  </item>
                  <item row="2" column="1">
                   <widget class="QDoubleSpinBox" name="rayBox">
                    <property name="enabled">
                     <bool>false</bool>
                    </property>
                    <property name="maximum">
                     <double>10000000000000000369475456880582265409809179829842688451922778552150543659347219597216513109705408327446511753687232667314337003349573404171046192448274432.000000000000000</double>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="soleilLayout">
               <item>
                <widget class="QLabel" name="posSoleilLayout">
                 <property name="text">
                  <string>Position du soleil</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="Line" name="lineSoleil">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QCheckBox" name="affPosCheckBox">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>Afficher la position</string>
               </property>
               <property name="checked">
                <bool>false</bool>
               </property>
               <property name="autoRepeatDelay">
                <number>300</number>
               </property>
               <property name="autoRepeatInterval">
                <number>100</number>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="XYZLayout">
               <item>
                <widget class="QLabel" name="XYZLabel">
                 <property name="text">
                  <string>Mesures XYZ</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="Line" name="lineXYZ">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QGroupBox" name="XYZTotGoupBox">
               <property name="title">
                <string>XYZ Total</string>
               </property>
               <layout class="QFormLayout" name="XYZTotalFormLayout">
                <item row="0" column="0">
                 <widget class="QLabel" name="labelXTot">
                  <property name="text">
                   <string>X</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QDoubleSpinBox" name="XTotBox">
                  <property name="maximum">
                   <double>10000000000000000369475456880582265409809179829842688451922778552150543659347219597216513109705408327446511753687232667314337003349573404171046192448274432.000000000000000</double>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="labelYTot">
                  <property name="text">
                   <string>Y</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QDoubleSpinBox" name="YTotBox">
                  <property name="maximum">
                   <double>10000000000000000369475456880582265409809179829842688451922778552150543659347219597216513109705408327446511753687232667314337003349573404171046192448274432.000000000000000</double>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QLabel" name="labelZTot">
                  <property name="text">
                   <string>Z</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1">
                 <widget class="QDoubleSpinBox" name="ZTotBox">
                  <property name="maximum">
                   <double>10000000000000000369475456880582265409809179829842688451922778552150543659347219597216513109705408327446511753687232667314337003349573404171046192448274432.000000000000000</double>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="XYZMasqueGroupBox">
               <property name="title">
                <string>XYZ avec masque</string>
               </property>
               <layout class="QFormLayout" name="XYZaMasqueFormLayout">
                <item row="0" column="0">
                 <widget class="QLabel" name="labelXMasque">
                  <property name="text">
                   <string>X</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QDoubleSpinBox" name="XMasqueBox">
                  <property name="maximum">
                   <double>10000000000000000369475456880582265409809179829842688451922778552150543659347219597216513109705408327446511753687232667314337003349573404171046192448274432.000000000000000</double>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="labelYMasque">
                  <property name="text">
                   <string>Y</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QLabel" name="labelZMasque">
                  <property name="text">
                   <string>Z</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QDoubleSpinBox" name="YMasqueBox">
                  <property name="maximum">
                   <double>10000000000000000369475456880582265409809179829842688451922778552150543659347219597216513109705408327446511753687232667314337003349573404171046192448274432.000000000000000</double>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1">
                 <widget class="QDoubleSpinBox" name="ZMasqueBox">
                  <property name="maximum">
                   <double>10000000000000000369475456880582265409809179829842688451922778552150543659347219597216513109705408327446511753687232667314337003349573404171046192448274432.000000000000000</double>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="rechargerButton">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>Corriger</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
            <zorder>XYZTotGoupBox</zorder>
            <zorder>XYZMasqueGroupBox</zorder>
            <zorder>verticalSpacer</zorder>
            <zorder>affMasqueBox</zorder>
            <zorder>affPosCheckBox</zorder>
            <zorder>avApBox</zorder>
            <zorder>rechargerButton</zorder>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>862</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="actionOuvrir"/>
    <addaction name="actionEnregistrer_sous"/>
   </widget>
   <addaction name="menuFichier"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionOuvrir">
   <property name="text">
    <string>Ouvrir</string>
   </property>
  </action>
  <action name="actionEnregistrer_sous">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Enregistrer sous...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
